Generator/Sources/NeedleFramework/Entry/PluginizedGenerator.swift

```diff
@@ -0,0 +1,31 @@
+//
```

Comment:
Should this one file live in a `Pluginized` subdirectory ? Seems silly for one file, but it's the convention we've used everywhere else.

Reply:
Yeah I think it'll be too many folders. Let's keep all the entry stuff in one folder for now, unless more shows up.